pkg/proxy/iptables/proxier_test.go

```diff
@@ -48,109 +48,6 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 )
 
-func checkAllLines(t *testing.T, table utiliptables.Table, save []byte, expectedLines map[utiliptables.Chain]string) {
-	chainLines := utiliptables.GetChainLines(table, save)
-	for chain, lineBytes := range chainLines {
-		line := string(lineBytes)
-		if expected, exists := expectedLines[chain]; exists {
-			if expected != line {
-				t.Errorf("getChainLines expected chain line not present. For chain: %s Expected: %s Got: %s", chain, expected, line)
-			}
-		} else {
-			t.Errorf("getChainLines expected chain not present: %s", chain)
-		}
-	}
-}
-
-func TestGetChainLines(t *testing.T) {
```

Comment:
I don't think we want to remove any of the existing unit tests in the iptables proxier.